pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -216,6 +216,9 @@ def work(self):
                                         if len(pokemon_to_transfer) == 0:
                                             raise RuntimeError(
                                                 'Trying to transfer 0 pokemons!')
+                                        # Add slight delay between capture & candy transfer #774
+                                        logger.log("Waiting briefly before transferring pokemon")
+                                        sleep(3)
```

Comment:
the sleep implementation does allow for randomness but I appear to be using it wrong. I'm going to add little function and update the pull request.